eng/scripts/SetTestPipelineVersion.ps1

```diff
@@ -0,0 +1,31 @@
+# Overides the project file and CHANGELOG.md for the template project using the next publishable version
+# This is to help with testing the release pipeline.
+. "${PSScriptRoot}\..\common\scripts\common.ps1"
+$latestTags = git tag -l "azure-sdk-template*beta*"
```

Comment:
Why `*beta*` as opposed to just `_*`?

Reply:
Am trying to not have to filter sort with the preview version. I don't assume we need to publish a GA version from the template pipeline.